app/services/claims_export_service.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+# Exports all claims that have been marked for needing export
+class ClaimsExportService
+
+  def initialize(claims_to_export: ClaimExport.includes(:claim),
+    claim_export_service: ClaimExportService, exported_file: ExportedFile)
+    self.claims_to_export = claims_to_export
+    self.claim_export_service = claim_export_service
+    self.exported_file = exported_file
+  end
+
+  # Exports everything
+  def export
+    Dir.mktmpdir do |dir|
+      export_claims to: dir
+      zip_files from: dir
+      persist_zip_file
+    end
+  ensure
+    remove_zip_if_exists
+  end
+
+  private
+
+  attr_accessor :claims_to_export, :claim_export_service, :exported_file
```

Comment:
I think you should be consistent where you put the accessors. In the file above ( app/services/claim_xml_import_service.rb) you put it at the end of the file. I prefer this version. But don't mind in general just stay consistent.